package/com.unity.formats.usd/Runtime/Scripts/Timeline/UsdRecorderBehaviour.cs

```diff
@@ -78,7 +78,7 @@ public class UsdRecorderBehaviour : PlayableBehaviour {
         Clip.Context.activePolicy = Clip.m_activePolicy;
         Clip.Context.exportMaterials = Clip.m_exportMaterials;
 
-        Clip.UsdScene.StartTime = currentTime;
+        Clip.UsdScene.StartTime = currentTime * 60;
```

Comment:
We should add a constant for the frame rate (e.g. rather than copying "60" around)

e.g. declare
`const int kExportFrameRate = 60;`

At the top of this file